docs/content/docs/custom-resource/overview.md

```diff
@@ -146,9 +148,51 @@ For Session clusters the operator only provides very basic management and monito
  - Monitor overall cluster health
  - Stop / Delete Session clsuter
 
+## FlinkSessionJob
+
+The FlinkSessionJob have the similar structure with FlinkDeployment with the following required fields:
+
+```
+apiVersion: flink.apache.org/v1beta1
+kind: FlinkSessionJob
+metadata:
+  name: basic-session-job-example
+spec:
+  deploymentName: basic-session-cluster
+  job:
+    jarURI: https://repo1.maven.org/maven2/org/apache/flink/flink-examples-streaming_2.12/1.14.4/flink-examples-streaming_2.12-1.14.4-TopSpeedWindowing.jar
+    parallelism: 4
+    upgradeMode: stateless
+```
+
+### FlinkSessionJob spec overview
+
+The spec contains the information to submit a session job to the session cluster. Mostly, it will define at least the following fields:
+
+ - deploymentName: The name of the target session cluster's CR
+ - job: The Spec of the Session Job
+
+The job spec is similar field with the application job, but the jarURI can support different source. It supports the http and filesystem based 
```

Comment:
wording: The job specification has the same structure in FlinkSessionJobs and FlinkDeployments, but in FlinkSessionJobs the jarUri can contain remote sources too.